lib/spack/llnl/util/cpu/detect.py

```diff
@@ -109,22 +110,42 @@ def sysctl(*args):
         'model name': sysctl('-n', 'machdep.cpu.brand_string')
     }
 
-    # Super hacky way to deal with slight representation differences
-    # Would be better to somehow consider these "identical"
-    if 'sse4.1' in info['flags']:
-        info['flags'] += ' sse4_1'
-    if 'sse4.2' in info['flags']:
-        info['flags'] += ' sse4_2'
-    if 'avx1.0' in info['flags']:
-        info['flags'] += ' avx'
-    if 'clfsopt' in info['flags']:
-        info['flags'] += ' clflushopt'
-    if 'xsave' in info['flags']:
-        info['flags'] += ' xsavec xsaveopt'
+    adjust_raw_flags(info)
```

Comment:
This function is also executed in `raw_info_dictionary`. Is this processing necessary?

Reply:
Nope, good catch! It's a leftover line from when I extracted a function from the code in there.

Reply:
Fixed